src/app/main/main.component.ts

```diff
@@ -316,16 +331,17 @@ export class MainComponent implements OnInit, OnDestroy {
           FROM (\n${sql.replace(/;\s*$/, '')}\n);`;
         return this.dataService.query(projectID, wrappedSQL, location);
       })
-      .then(({ columns, columnNames, rows, stats, totalRows }) => {
+      .then(({ columns, columnNames, rows, stats, totalRows, pageToken, jobId }) => {
         this.columns = columns;
         this.columnNames = columnNames;
         this.geoColumnNames = geoColumns.map((f) => f.name)
-        this.rows = rows;
+	this.rows = rows;                                                                                      
```

Comment:
Whitespace issues here.^

Reply:
Done.

Reply:
Done

Reply:
Rest of the file looks good now, but it seems like this is still indented oddly?

Got a semicolon missing on the line above it, maybe that's throwing off the formatting.

Reply:
yeah. fixed it for realz this time. :)